substrate-node/node/src/service.rs

```diff
@@ -123,6 +123,15 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
 		})?;
 
 	if config.offchain_worker.enabled {
```

Comment:
Offchain workers can be used for a variety of things, imo the name `offchain_worker.enabled` is not very good, please change it to a functional name like related to the fact that the tft-price is constantly updated.

Reply:
https://github.com/paritytech/substrate/blob/b535922ae71c8cd68741ed26d042ba159725fddd/client/service/src/config.rs#L107